docker-compose/mariadb/docker-compose.yaml

```diff
@@ -8,7 +8,7 @@ volumes:
 services:
   mariadb:
     # (Recommended) replace "latest" with specific version
-    image: mariadb:latest
+    image: docker.io/library/mariadb:latest
```

Comment:
pin to version this can break a lot of stuff with al :latest

Reply:
@jwklijnsma That's not the scope of this PR.

Reply:
@jwklijnsma You might also follow up reading [Best practices for creating pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/best-practices-for-pull-requests#best-practices-for-creating-pull-requests). Keep PRs small. Focus on **one** topic. If the PR is about solving problem X, it shouldn't also deal with problem Y and Z. That's up to different PRs dealing with those.